docs/api-reference/extensions/collide-extension.md

```diff
@@ -5,8 +5,14 @@ The `CollideExtension` allows layers to hide features which overlap with other f
 
 To use this extension on a layer, add the `CollideExtension` to the layer's `extensions` prop.
 
-<!-- TODO: Codepen demo -->
 <div style={{position:'relative',height:450}}></div>
+<div style={{position:'absolute',transform:'translateY(-450px)',paddingLeft:'inherit',paddingRight:'inherit',left:0,right:0}}>
```

Comment:
Nice demo, I think we could make it prettier though. Add a point layer for the nodes?

```js
const pointLayer = new ScatterplotLayer({
    id: 'points',
    data: Object.values(nodes),
    getPosition: node => [node.x, node.y],
    getRadius: getIconSize,
    radiusScale: 0.05,
    radiusUnits: 'pixels',
    lineWidthUnits: 'pixels',
    stroked: true,
    getFillColor: [222, 222, 222],
    getLineColor: [100, 100, 100]
  });
```
<img width="790" alt="Screenshot 2023-02-23 at 10 11 17" src="https://user-images.githubusercontent.com/453755/220864553-6324cf4b-3309-4f3b-9a4a-bdd87f26ee62.png">